parser.go

```diff
@@ -0,0 +1,137 @@
+package voyager
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var noTxPrefix = regexp.MustCompile("^\\s*--\\s+(NO_TRANSACTION)")
+var migrationDirection = regexp.MustCompile("\\.(up|down)\\.")
```

Comment:
Ditto for the above suggestion.

```suggestion
var migrationDirection = regexp.MustCompile(`\.(up|down)\.`)
```